### PR TITLE
np2 0.84 (new formula)

### PR DESCRIPTION
**Pull request by tomyun**
[Neko Project II](http://www.yui.ne.jp/np2/) is an emulator for [PC-9801](https://en.wikipedia.org/wiki/NEC_PC-9801) series computers. There were many Japanese games exclusively released for this platform in 80~90s.

The project has been recently updated to support Mac OS X builds. A few changes for brewing include:

- Use brewed `SDL2`, `SDL2_ttf` paths in Xcode project file
- Turn on TTF font by default (otherwise, garbles fonts displayed)
- Always use current working directory (otherwise, font cache saved in `/usr/local/`)

FYI, [some](http://pkgsrc.se/emulators/xnp2) Unix distros have a package for [xnp2](https://github.com/nonakap/xnp2) which is an X11/GTK port.

### CODE REVIEW

**Comment by tomyun** [comment]
`test` block that runs `np2 -h` is removed. It returns with exit code 1 after displaying help message.

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

File: np2.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      # Always use current working directory
+      inreplace "np2sdl2/main.m", "[pstrBundlePath UTF8String]", '"./"'
+
+      xcodebuild "SYMROOT=build"
+      bin.install "build/Release/np2sdl2.app/Contents/MacOS/np2sdl2" => "np2"
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    A Japanese TTF file named `default.ttf` should be in the working directory.
```
Comment (mistydemeo) [inline np2.rb]: Is there a source of this we can link to?

Comment (tomyun) [inline np2.rb]: I tested with Osaka font from OS X: `cp /Library/Fonts/Osaka.ttf ./default.ttf`.
It looked okay, but not sure what people usually have.

**Comment by mistydemeo** [comment]
Thanks!

**Pull request closed without merging**